flex-config/index.js

```diff
@@ -1,11 +1,12 @@
 const axios = require("axios");
 const dotenv = require("dotenv");
-const { promises: Fs } = require('fs')
+const fs = require('fs');
```

Comment:
If moving to the sync version of `fs`, we need to refactor the existing calls to `fs` to use that instead of await and try/catch. Otherwise, we should go back to the promises version.